incubator/zookeeper/Chart.yaml

```diff
@@ -1,10 +1,12 @@
 name: zookeeper
 home: https://zookeeper.apache.org/
-version: 0.1.1
+version: 0.1.2
```

Comment:
Lets also move this to the stable folder. The only additional thing to do is to add a NOTES.txt file that tells users how they can connect to the ZK cluster from both inside and outside the cluster. The Jenkins chart has a good example.